source/_static/js/main.js

```diff
@@ -504,4 +504,12 @@ window.addEventListener("DOMContentLoaded", (event) => {
       });
     }
   })();
-});
\ No newline at end of file
+});
+
+
+window.addEventListener("hashchange", function() {
+  document.getElementById((document.getElementById(window.location.hash.substring(1)))
```

Comment:
So I think we also need this somewhere above line 507, so that if someone links directly to a 'hidden' anchor item, it gets picked up.

This only triggers if the hash _changes_ on the page, not if the anchor already exists.